lib/src/pylons_wallet/pylons_wallet_impl.dart

```diff
@@ -110,9 +110,13 @@ class PylonsWalletImpl implements PylonsWallet {
   }
 
   @override
-  Future<SDKIPCResponse> getTrades() async {
+  Future<SDKIPCResponse<List<Trade>>> getTrades() async {
     return Future.sync(() async {
-      return await _dispatch(Strings.GET_TRADES, '');
```

Comment:
Get trades should be based on the creator.

Reply:
@kjawadDeveloper2 , Can you review the changes please?